### PR TITLE
[2017-Brasilia] WIP: 2017 Brasilia kickoff

### PR DESCRIPTION
**Pull request by adrianovieira**
don't merge yet

### CODE REVIEW

**Comment by MikeRosTX** [comment]
Set the **do-not-merge** flag as requested by @adrianovieira.
Since all 4 organizers listed in the PR are the same as last year, we created the new email **organizers-brasilia-2017@devopsdays.org** from last year's list.  Slack should already be good to go.

**Comment by MikeRosTX** [comment]
@adrianovieira let me know when you're ready for me to merge and get it posted.

**Comment by MikeRosTX** [comment]
@adrianovieira let me know when you're ready for me to merge and get it posted.

File: data/events/2017-brasilia.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ description: "Devopsdays is coming to Brasília!" # Edit this to suit your prefe
 ga_tracking_id: "UA-83133994-2" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
 
 # All dates are in unquoted 2017-MM-DD, like this:   variable: 2016-01-05
-startdate:  # The start date of your event. Leave blank if you don't have a venue reserved yet.
-enddate:  # The end date of your event. Leave blank if you don't have a venue reserved yet.
+startdate: 2017-11-07 # The start date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (MikeRosTX) [inline data/events/2017-brasilia.yml]: @adrianovieira Deixe a data em branco se você não tem um lugar reservado para essa data.

**Comment by adrianovieira** [comment]
@MikeRosTX we need to be sure that the email `organizers-brasilia-2017@devopsdays.org` has only these four members for now. Is that right?

**Comment by MikeRosTX** [comment]
@adrianovieira Yes the following 4 organizers are setup in email

adriano.svieira@gmail.com 
gutocarvalho@instruct.com.br    
laurosn@gmail.com    
tacianotres@gmail.com 

But you need to remove Start and End date of the event if you don't have a venue location.

**Comment by MikeRosTX** [comment]
@adrianovieira when you have a confirmed venue location, create a new PR and add the date below to the **2017-brazilia.yml**  But we are not allowing anyone to have dates set, if you don't have a signed confirmed contract with the venue.

# All dates are in unquoted 2017-MM-DD, like this:   variable: 2016-01-05
startdate: 2017-11-07 # The start date of your event. Leave blank if you don't have a venue reserved yet.
enddate: 2017-11-08 # The end date of your event. Leave blank if you don't have a venue reserved yet.

# Leave CFP dates blank if you don't know yet, or set all three at once.
cfp_date_start: 2017-03-25 # start accepting talk proposals.
cfp_date_end: 2017-06-30 # close your call for proposals.
cfp_date_announce: 2017-07-17 # inform proposers of status

cfp_open: "true"

**Pull request merged**